java/dynamodb-tools/src/main/java/sleeper/dynamodb/tools/DynamoDBUtils.java

```diff
@@ -73,4 +77,22 @@ public static void configureTimeToLive(AmazonDynamoDB dynamoDB, String tableName
                 ));
         LOGGER.info("Configured TTL on field {}", expiryField);
     }
+
+    public static List<Map<String, AttributeValue>> doScanWithPagination(AmazonDynamoDB dynamoDB, ScanRequest scanRequest) {
```

Comment:
We probably want to avoid gathering all the items into a list if it's enough that we need to page through them.

It would be possible, though fiddly, to implement this by generating a Stream. An alternative would be to use a do/while loop everywhere a scan is done, like in the DynamoDB documentation: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ScanJavaDocumentAPI.html

The name might read better without "do" at the start.